test/e2e/events_test.go

```diff
@@ -62,11 +61,11 @@ var _ = Describe("Podman events", func() {
 		result := podmanTest.Podman([]string{"events", "--stream=false", "--filter", "event=start"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(len(result.OutputToStringArray()) >= 1)
+		Expect(len(result.OutputToStringArray())).To(BeNumerically(">=", 1), "Number of events")
 	})
 
 	It("podman events with an event filter and container=cid", func() {
-		Skip("Does not work on v2")
+		// FIXME: Why was this skipped? It works fine on my laptop
```

Comment:
Someone please address this.

Reply:
I have no idea, this should work in all cases AFAICT. I would say the `SkipIfNotFedora()` should be removed as well.

Reply:
Done. I will resubmit with both `Skip()s` removed.